googletest/include/gtest/gtest.h

```diff
@@ -2499,6 +2499,20 @@ inline int RUN_ALL_TESTS() {
   return ::testing::UnitTest::GetInstance()->Run();
 }
 
+#ifdef ARDUINO
+inline void gtest_setup() {
+  // Since Arduino doesn't have a command line, fake out the argc/argv arguments
+  int argc = 1;
```

Comment:
Let us Inot define these functions in the header. Too much noise.

Also, there is nothing interesting about this function.
If the user has to insert code to initialize GoogleTest, they might as well call testing::InitGoogleTest themselves.

At this point I think it might be simpler to provide an overload of InitGoogleTest that doesn't require argv/argc.

Reply:
I like that idea better.  I pushed no argument versions of the init functions and cleaned up where appropriate.